src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala

```diff
@@ -275,6 +304,18 @@ object FileService {
     }
   }
 
+  implicit class CustomMetadataFieldsHelper(fields: Seq[FilepropertyRow]) {
```

Comment:
Maybe rename the implicit class and functions to reflect this is handling `FilepropertyRow`s:

`implicit class FilePropertyRowsHelper(rows: Seq[FilepropertyRow]) ... etc ...`

Reply:
Yeah, sorry, I forgot to change the name.